README.md

```diff
@@ -1,2 +1,23 @@
 # parse-key-value-pair
-Parse KEY=value strings with ease
+
+Parse a string like `KEY=value` into `[key, value]`
+
+## Installation
+
+```sh
+yarn add parse-key-value-pair
```

Comment:
Personally, I usually do `npm install --save ...` since the people who use `yarn` already know how to use it, but people not familiar with `yarn` might not know that you can use `npm install` instead...